heartbeat/aws-vpc-move-ip

```diff
@@ -261,10 +265,17 @@ ec2ip_monitor() {
 ec2ip_drop() {
 	cmd="ip addr delete ${OCF_RESKEY_ip}/32 dev $OCF_RESKEY_interface"
 	ocf_log debug "executing command: $cmd"
-	$cmd
+	output=$($cmd 2>&1)
 	rc=$?
+
 	if [ "$rc" -gt 0 ]; then
-		ocf_log warn "command failed, rc $rc"
+		if [ "$__OCF_ACTION" = "start" ]; then
+			level=debug
+		else
+			level=warn
+		fi
+		ocf_log "$level" "command failed, rc $rc"
+		ocf_log "$level" "output/error: $output"
```

Comment:
There should be no stdout from `ip addr del`; only `stderr` if there's a problem. So I figure, no need to print this unless `rc > 0`.

Reply:
Unless you do it at debug-level.

Reply:
Sure, and if you'd prefer that, I'll update it.

I don't have a strong opinion. I settled on this because it put all the logging stuff in one place, and because an rc=0 never seems to generate any stdout or stderr (so it generates a logging line that only says "output/error:" every time).

Reply:
I'm fine either way.